etc/di.xml

```diff
@@ -1035,4 +1035,20 @@
     <type name="Magento\Checkout\Api\GuestPaymentInformationManagementInterface">
         <plugin name="AdyenGuestPaymentInformationManagementAddPaymentInfo" type="Adyen\Payment\Plugin\GuestPaymentInformationManagement" sortOrder="10"/>
     </type>
+
+    <!--Notifications overview-->
+    <type name="Magento\Framework\View\Element\UiComponent\DataProvider\CollectionFactory">
+        <arguments>
+            <argument name="collections" xsi:type="array">
+                <item name="adyen_notificationsoverview_data_source" xsi:type="string">Adyen\Payment\Model\ResourceModel\Notification\UiComponentCollection</item>
```

Comment:
shall we add a _ after notifications so it becomes more readable ? adyen_notifications_overview_data_source

Reply:
Done, I also changed the main controller name accordingly